### ISSUE TITLE
V4 Bug: CORE_RETURN_HTTPS

### ISSUE THREAD

**Issue by iiTranquility** [dicedtomatoreal/typex#593]
### What happened?

When using either CORE_RETURN_HTTPS and, as well as or CORE_RETURN_HTTPS_URLS (as it is in the code), I get an error either way when uploading via Sharex.

### Version

latest (ghcr.io/diced/zipline or ghcr.io/diced/zipline:latest)

### Zipline Logs

```shell
Unable to parse response content returned from host (domain.iitranq.co.uk).

System.ArgumentException: Can not convert Object to String.
   at Newtonsoft.Json.Linq.JToken.op_Explicit(JToken value)
   at ShareX.UploadersLib.ShareXCustomUploaderSyntaxParser.CallFunction(String functionName, String[] parameters)
   at ShareX.UploadersLib.ShareXSyntaxParser.Parse(String text, Boolean isFunction, Int32 startPosition, Int32& endPosition)
   at ShareX.UploadersLib.ShareXSyntaxParser.Parse(String text, Boolean isFunction, Int32 startPosition, Int32& endPosition)
   at ShareX.UploadersLib.ShareXSyntaxParser.Parse(String text)
   at ShareX.UploadersLib.CustomUploaderItem.ParseResponse(UploadResult result, ResponseInfo responseInfo, UploaderErrorManager errors, CustomUploaderInput input, Boolean isShortenedURL)
   at ShareX.UploadersLib.CustomUploaderItem.TryParseResponse(UploadResult result, ResponseInfo responseInfo, UploaderErrorManager errors, CustomUploaderInput input, Boolean isShortenedURL)
```

**Comment by diced** [comment]
I think this might be a problem with the sharex generator if you used that, since that looks like an error from ShraeX. It would be more helpful if you provided the generated .sxcu instead :)

**Comment by iiTranquility** [comment]
https://i.iitranq.co.uk/v73yvlmBKc.txt

It's the sharex sxcu, it downloaded as a sxcu.txt on mobile for some reason but it works for the most part

**Comment by diced** [comment]
ah it's the `{json:files[0]}` which should actually be `{json:files[0].url}`. fixed in db28b06791b91351a8fd0dd3f260f6aa524eebce
